### PR TITLE
Updates node_modules based on the latest tools/package.json

### LINKED ISSUE #69: Update node_modules folder due to removed prettier dependency
Waiting on https://github.com/denoland/deno/pull/6682

### PR DESCRIPTION
**Pull request by dsherret**
Updated this folder based on https://github.com/denoland/deno/blob/master/tools/package.json

Closes #69

### CODE REVIEW

File: node_modules/typescript/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "typescript",
     "author": "Microsoft Corp.",
     "homepage": "https://www.typescriptlang.org/",
-    "version": "3.9.2",
+    "version": "3.6.3",
```
Comment (trivikr) [inline node_modules/typescript/package.json:5]: This downgrades typescript as old version (v3.6.3) is listed in https://github.com/denoland/deno/blob/69552b1defcd77748de87fc4c9311b1baec1361d/tools/package.json#L10

Comment (dsherret) [reply node_modules/typescript/package.json:5]: Hmmm... I wonder how this was generated in past commits. Obviously something is off here.

Comment (trivikr) [reply node_modules/typescript/package.json:5]: I couldn't find documented process to update deno_third_party modules in this repo.
I manually updated typescript to v3.9.7 (copying contents from yarn install in another folder) in PR https://github.com/denoland/deno_third_party/pull/75

**Comment by dsherret** [comment]
I'm going to close this due to https://github.com/denoland/deno_third_party/pull/72#discussion_r460473836

I'm not sure what the proper resolution is.

**Pull request closed without merging**